apps/fluent-tester/src/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -394,14 +405,15 @@ const CustomCallout: React.FunctionComponent = () => {
 
       {showCustomizedCallout && (
         <Callout
+          vibrant={true}
           anchorRect={myRect}
           onDismiss={onDismissCustomizedCallout}
           onShow={onShowCustomizedCallout}
           accessibilityLabel="Customized Callout"
           accessibilityRole="alert"
           accessibilityOnShowAnnouncement="Be informed that a customized callout has been opened."
         >
-          <View style={{ padding: 20, borderWidth: 2, borderColor: 'black' }}>
+          <View style={{ padding: 20, borderWidth: 2, borderColor: 'black', backgroundColor: theme.colors.neutralBackground1 }}>
```

Comment:
Is `neutralBackground1` the correct default color for a callout? I think I recall in some design spec somewhere that we wanted `neutralBackground4` by default, to better align with NSMenu/NSPopover.

Reply:
Good callout (no pun intended), I just chose a random color.